### PR TITLE
Update pagination to be responsive

### PR DESCRIPTION
**Pull request by antsand**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/components/common/Pagination/showingText.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { ListPageNumberProps } from './interfaces';
+
+const ShowingText = (props: ListPageNumberProps) => {
+  return (
+    <>
+      <span>
+        Showing {(props.currentPage - 1) * props.pageSize + 1} to{' '}
+        {props.totalEntries < props.pageSize * props.currentPage
+          ? props.totalEntries
+          : props.currentPage * props.pageSize}{' '}
```
Comment (binggaofw) [inline src/frontend/src/components/common/Pagination/showingText.tsx:10]: are we using `{' '}` to add space? @kushal-arora-fw  I thought we do not use it

File: src/frontend/src/components/common/Pagination/index.tsx [hunk 2]
```diff
@@ -4,6 +4,11 @@ import {
   faAngleLeft,
   faAngleRight,
```
Comment (binggaofw) [inline src/frontend/src/components/common/Pagination/index.tsx:5]: these import statements does not use anymore by this file, i think can clean up

File: src/frontend/src/components/common/Pagination/rowsPerPage.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+
+export interface rowsPerPageProps {
```
Comment (binggaofw) [inline src/frontend/src/components/common/Pagination/rowsPerPage.tsx:4]: interface first letter must be Upper case, other wise you can not build successful

**Review by binggaofw — DISMISSED** [review]

**Comment by kushal-arora-fw** [comment]
Note: Please DO NOT MERGE until Lint is fixed.

File: src/frontend/src/components/common/Pagination/button.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { PaginationButtonProps, Direction } from './interfaces';
```
[the remark this answers is not in the document]
Comment (kushal-arora-fw) [reply src/frontend/src/components/common/Pagination/button.tsx:2]: `pagintation-direction.enum.ts` or simply like interfaces - `enum.ts` ?

**Comment by kushal-arora-fw** [comment]
Please run linter / formatter on `showingText.tsx` - that should fix linting error. @antsand

**Review by binggaofw — APPROVED** [review]

**Review by kushal-arora-fw — APPROVED** [review]

**Pull request merged**